lib/CtrlxCore.js

```diff
@@ -807,7 +807,7 @@ class CtrlxCore {
       }
 
       // The subscription is encapsulated in a separate object.
-      let sub = new CtrlxDatalayerSubscription(this._hostname, this._servername, this._tokenType + ' ' + this._token, paths, publishIntervalMs);
+      let sub = new CtrlxDatalayerSubscription(this._hostname, this._port, this._servername, this._tokenType + ' ' + this._token, paths, publishIntervalMs);
```

Comment:
1. Is there any option to set the port in the Editor by the user? I cannot find any either on the subscription or the device config node. 
2. How can I setup the tests to be executed using a different port than 443?

Reply:
1. You can set the port by appending in the hostname. E.g. "192.168.1.1:8443"
2. You can change the hostname and port for the unit test by setting the environment variable TEST_HOSTNAME before the test. E.g. "export TEST_HOSTNAME=localhost:1234"